tests/testing_utils.py

```diff
@@ -63,16 +63,16 @@ def make_train_job(model_json, initial_weights, hyperparams, \
     )
     return train_job
 
-def make_serialized_job(session_filepath=None):
+def make_serialized_job(raw_filepath):
     """
     TODO: Optimizer should handle transform/split so that datapoint_count
     and session_filepath don't have to be attached manually.
     """
-    initialize_job = make_initialize_job(make_model_json())
-    if session_filepath:
-        datapoint_count = count_datapoints(session_filepath)
-        initialize_job.datapoint_count = datapoint_count
-        initialize_job.session_filepath = session_filepath
+    initialize_job = make_initialize_job(make_model_json(), raw_filepath)
+    # if session_filepath:
+    #     datapoint_count = count_datapoints(session_filepath)
+    #     initialize_job.datapoint_count = datapoint_count
+    #     initialize_job.session_filepath = session_filepath
```

Comment:
You should just go ahead and remove this block. Also, please remove the TODO in the docstring/

Reply:
Done